rest/specs/src/main/filtered-resources/openapi/openapi.yaml

```diff
@@ -194,32 +194,19 @@ components:
       description: Transformation request.
       type: object
       required:
-        - id
-        - stagedLocation
-        - productLocation
+        - location
         - mimeType
         - bytes
-        - callbackUrl
       properties:
-        id:
-          type: string
-          description: The unique identifier for a request.
-          minLength: 1
-          maxLength: 255
-          pattern: '^[a-zA-Z0-9]*$'
-          example: 30f14c6c1fc85cba12bfd093aa8f90e3
-        stagedLocation:
-          type: string
-          description: The temporary location of the ingested product.
-          minLength: 1
-          maxLength: 2048
-          example: https://bucket.s3.amazonaws.com
-        productLocation:
+        location:
           type: string
-          description: The final location of the ingested product.
+          format: uri
+          description: >
+            The URI of the ingested product. This URI must support HTTP GET at location/{productId} to retrieve
+            the product and HTTP PUT with attachment at location/{productId}/{metadataType} to store metadata.
```

Comment:
This should be POST, not PUT.

Reply:
Since transformation knows exactly what the URI is and is not asking the callee to return it, we agreed to use `PUT` instead of `POST` here.